src/relay/backend/compile_engine.cc

```diff
@@ -98,8 +98,10 @@ Array<IndexExpr> GetShape(const Array<IndexExpr>& shape) {
 // Get schedule from functor.
 class ScheduleGetter : public backend::MemoizedExprTranslator<Array<te::Tensor>> {
  public:
-  explicit ScheduleGetter(Target target)
-      : target_(target), device_copy_op_(Op::Get("device_copy")) {}
+  explicit ScheduleGetter(Target target, bool use_auto_schedule)
```

Comment:
Can we just get the flag from PassContext::Current() instead of passing it around?

Reply:
Good point. All added arguments are removed.